pkg/metrics/metrics.go

```diff
@@ -212,6 +215,13 @@ var (
 		Help:      "Number of times a policy import has failed",
 	})
 
+	// PolicyEndpointStatus is the number of endpoints with policy labeled by enforcement type
+	PolicyEndpointStatus = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: Namespace,
+		Name:      "policy_endpoint_enforcement_status",
```

Comment:
Rename to follow https://prometheus.io/docs/practices/naming/.
This counts endpoints, so I'd rename this into `policy_enforcement_status_endpoints_total`.

Reply:
Only counters should use `_total`, not gauges. Let's please just use promlint and it will warn about all of this.